private/buf/cmd/buf/command/alpha/protoc/internal/protoc-gen-plugin-config/main.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020-2023 Buf Technologies, Inc.
```

Comment:
This shouldn't be in `alpha/protoc` - this is for the `buf alpha protoc` command, this should be somewhere for generate